src/insights_client/metrics.py

```diff
@@ -31,6 +35,43 @@ def _proxy_settings(rhsm_config):
     return {"https": proxy}
 
 
+def _is_offline(cfg):
+    '''
+    Determine whether offline mode was specified in
+    either the config or the CLI
+
+    :param cfg: RawConfigParser with the
+                insights-client config file read into it
+
+    Returns True if offline, False if not offline
+    '''
+    def _boolify(v):
+        if v.lower() == 'true':
+            return True
+        elif v.lower() == 'false':
+            return False
+        else:
+            return v
```

Comment:
This can live outside the __is_offline function_, so it is not redefined with every call.

```suggestion
```